script.js

```diff
@@ -0,0 +1,54 @@
+
+let slidePosition = 0;
+const slides = document.getElementsByClassName('carousel-item');
+const totalSlides = slides.length;
+
+const nextBtn = document.getElementById('carousel_button-next');
+nextBtn.addEventListener('click', moveToNextSlide)
+
+const prevBtn = document.getElementById('carousel_button-previous');
```

Comment:
Generally, there is no reason to shorten variable names, full words read better.
prevBtn 
previousButton
backButton